src/test/java/uk/gov/hmcts/reform/roleassignment/domain/service/common/PersistenceServiceTest.java

```diff
@@ -573,6 +598,46 @@ void shouldReturnEmptyListOfRoleAssignmentRecords()  {
         assertEquals(roleAssignmentList.size(),Integer.valueOf(0));
     }
 
+    @Test
+    void shouldReturnEmptyListOfRoleAssignmentRecordsWithRecordsPresentExistingFlagFalse() throws IOException {
```

Comment:
Please rename the method.

Reply:
Changed to shouldGetEmptyListForExistingFlagFalse.